### PR TITLE
Issue #2721 xpaas addon version doesn't match OpenShift version

### LINKED ISSUE #2721: xpaas addon version doesn't match OpenShift version
Starting minishift with:
```
minishift start --vm-driver=virtualbox --openshift-version=v3.10.0
```

And apply the xpaas addon, installs the imagestreams from v3.9:
```
$ minishift addons apply xpaas
-- Applying addon 'xpaas':.........................................................................................................................
XPaaS OpenShift imagestream and templates installed
See https://github.com/openshift/openshift-ansible/tree/release-3.9/roles/openshift_examples/files/examples/v3.9
```

### PR DESCRIPTION
**Pull request by agajdosi**
Fixes #2721

Steps to test the pull request

  1. minishift start 
  2. minishift addons apply xpaas
  3. try

  3. minishift start --openshift-version v3.9.0
  4. minishift addons apply --addon-env OPENSHIFT_VERSION=3.9
  5. try

### CODE REVIEW

**Comment by agajdosi** [comment]
retest this please

**Comment by praveenkumar** [comment]
@agajdosi We are explicitly told long back to remove eap6.x from the template and from the image stream, if you see 3.9 one those are not part of addon.

File: addons/xpaas/v3.10/xpaas-streams/eap64-image-stream.json [hunk 1]
```diff
@@ -0,0 +1,194 @@
+{
```
Comment (praveenkumar) [inline addons/xpaas/v3.10/xpaas-streams/eap64-image-stream.json]: @agajdosi We are explicitly told long back to remove eap6.x from the template and from the image stream, if you see 3.9 one those are not part of addon.

Comment (agajdosi) [reply addons/xpaas/v3.10/xpaas-streams/eap64-image-stream.json]: @praveenkumar Thanks mate, completely missed that. Will remove that.

**Comment by agajdosi** [comment]
@praveenkumar eap 6.4 stream and templates removed.

**Comment by praveenkumar** [comment]
retest this please.

**Comment by amitkrout** [comment]
@agajdosi Can you please enable the xpaas addons along with the ```integration_pr``` just to check if its working properly in CI.

**Comment by praveenkumar** [comment]
CI failure is not related to this PR.

**Comment by praveenkumar** [comment]
Tried and able to see the image stream and templates for xpass.

```
 oc get is -n openshift
NAME                                           DOCKER REPO                                                              TAGS                           UPDATED
dotnet                                         172.30.1.1:5000/openshift/dotnet                                         2.0,latest                     2 minutes ago
fis-java-openshift                             172.30.1.1:5000/openshift/fis-java-openshift                                                            
fis-karaf-openshift                            172.30.1.1:5000/openshift/fis-karaf-openshift                                                           
httpd                                          172.30.1.1:5000/openshift/httpd                                          2.4,latest                     2 minutes ago
java                                           172.30.1.1:5000/openshift/java                                                                          
jboss-amq-62                                   172.30.1.1:5000/openshift/jboss-amq-62                                                                  
jboss-amq-63                                   172.30.1.1:5000/openshift/jboss-amq-63                                   1.1,1.2,1.3 + 1 more...        1 second ago
jboss-datagrid65-client-openshift              172.30.1.1:5000/openshift/jboss-datagrid65-client-openshift              1.0,1.1                        13 seconds ago
jboss-datagrid65-openshift                     172.30.1.1:5000/openshift/jboss-datagrid65-openshift                                                    
jboss-datagrid71-client-openshift              172.30.1.1:5000/openshift/jboss-datagrid71-client-openshift              1.0                            7 seconds ago
jboss-datagrid71-openshift                     172.30.1.1:5000/openshift/jboss-datagrid71-openshift                                                    
jboss-datagrid72-openshift                     172.30.1.1:5000/openshift/jboss-datagrid72-openshift                                                    
jboss-datavirt63-driver-openshift              172.30.1.1:5000/openshift/jboss-datavirt63-driver-openshift                                             
jboss-datavirt63-openshift                     172.30.1.1:5000/openshift/jboss-datavirt63-openshift                                                    
jboss-decisionserver62-openshift               172.30.1.1:5000/openshift/jboss-decisionserver62-openshift                                              
jboss-decisionserver63-openshift               172.30.1.1:5000/openshift/jboss-decisionserver63-openshift                                              
jboss-decisionserver64-openshift               172.30.1.1:5000/openshift/jboss-decisionserver64-openshift                                              
jboss-eap70-openshift                          172.30.1.1:5000/openshift/jboss-eap70-openshift                                                         
jboss-eap71-openshift                          172.30.1.1:5000/openshift/jboss-eap71-openshift                                                         
jboss-fuse70-console                           172.30.1.1:5000/openshift/jboss-fuse70-console                                                          
jboss-fuse70-eap-openshift                     172.30.1.1:5000/openshift/jboss-fuse70-eap-openshift                                                    
jboss-fuse70-java-openshift                    172.30.1.1:5000/openshift/jboss-fuse70-java-openshift                                                   
jboss-fuse70-karaf-openshift                   172.30.1.1:5000/openshift/jboss-fuse70-karaf-openshift                                                  
jboss-processserver63-openshift                172.30.1.1:5000/openshift/jboss-processserver63-openshift                                               
jboss-processserver64-openshift                172.30.1.1:5000/openshift/jboss-processserver64-openshift                                               
jboss-webserver30-tomcat7-openshift            172.30.1.1:5000/openshift/jboss-webserver30-tomcat7-openshift                                           
jboss-webserver30-tomcat8-openshift            172.30.1.1:5000/openshift/jboss-webserver30-tomcat8-openshift                                           
jboss-webserver31-tomcat7-openshift            172.30.1.1:5000/openshift/jboss-webserver31-tomcat7-openshift                                           
jboss-webserver31-tomcat8-openshift            172.30.1.1:5000/openshift/jboss-webserver31-tomcat8-openshift                                           
jenkins                                        172.30.1.1:5000/openshift/jenkins                                        1,2,latest                     2 minutes ago
mariadb                                        172.30.1.1:5000/openshift/mariadb                                        10.1,10.2,latest               About a minute ago
mongodb                                        172.30.1.1:5000/openshift/mongodb                                        2.4,2.6,3.2 + 2 more...        About a minute ago
mysql                                          172.30.1.1:5000/openshift/mysql                                          5.5,5.6,5.7 + 1 more...        About a minute ago
nginx                                          172.30.1.1:5000/openshift/nginx                                          latest,1.10,1.12 + 1 more...   About a minute ago
nodejs                                         172.30.1.1:5000/openshift/nodejs                                         0.10,4,6 + 2 more...           About a minute ago
perl                                           172.30.1.1:5000/openshift/perl                                           5.16,5.20,5.24 + 1 more...     About a minute ago
php                                            172.30.1.1:5000/openshift/php                                            7.1,latest,5.5 + 2 more...     About a minute ago
postgresql                                     172.30.1.1:5000/openshift/postgresql                                     9.2,9.4,9.5 + 2 more...        About a minute ago
python                                         172.30.1.1:5000/openshift/python                                         2.7,3.3,3.4 + 3 more...        About a minute ago
redhat-openjdk18-openshift                     172.30.1.1:5000/openshift/redhat-openjdk18-openshift                                                    
redhat-sso70-openshift                         172.30.1.1:5000/openshift/redhat-sso70-openshift                                                        
redhat-sso71-openshift                         172.30.1.1:5000/openshift/redhat-sso71-openshift                                                        
redhat-sso72-openshift                         172.30.1.1:5000/openshift/redhat-sso72-openshift                                                        
redis                                          172.30.1.1:5000/openshift/redis                                          3.2,latest                     About a minute ago
rhdm70-decisioncentral-openshift               172.30.1.1:5000/openshift/rhdm70-decisioncentral-openshift                                              
rhdm70-kieserver-openshift                     172.30.1.1:5000/openshift/rhdm70-kieserver-openshift                                                    
rhpam70-businesscentral-indexing-openshift     172.30.1.1:5000/openshift/rhpam70-businesscentral-indexing-openshift                                    
rhpam70-businesscentral-monitoring-openshift   172.30.1.1:5000/openshift/rhpam70-businesscentral-monitoring-openshift                                  
rhpam70-businesscentral-openshift              172.30.1.1:5000/openshift/rhpam70-businesscentral-openshift                                             
rhpam70-controller-openshift                   172.30.1.1:5000/openshift/rhpam70-controller-openshift                                                  
rhpam70-kieserver-openshift                    172.30.1.1:5000/openshift/rhpam70-kieserver-openshift                                                   
rhpam70-smartrouter-openshift                  172.30.1.1:5000/openshift/rhpam70-smartrouter-openshift                                                 
ruby                                           172.30.1.1:5000/openshift/ruby                                           2.0,2.2,2.3 + 3 more...        About a minute ago
wildfly                                        172.30.1.1:5000/openshift/wildfly                                        11.0,12.0,8.1 + 4 more...      About a minute ago

$ oc get templates -n openshift
NAME                                            DESCRIPTION                                                                        PARAMETERS        OBJECTS
amq62-basic                                     Application template for JBoss A-MQ brokers. These can be deployed as standal...   11 (4 blank)      6
amq62-persistent                                Application template for JBoss A-MQ brokers. These can be deployed as standal...   13 (4 blank)      8
amq62-persistent-ssl                            An example JBoss A-MQ application. For more information about using this temp...   18 (6 blank)      12
amq62-ssl                                       Application template for JBoss A-MQ brokers. These can be deployed as standal...   16 (6 blank)      10
amq63-basic                                     Application template for JBoss A-MQ brokers. These can be deployed as standal...   11 (4 blank)      6
amq63-persistent                                An example JBoss A-MQ application. For more information about using this temp...   13 (4 blank)      8
amq63-persistent-ssl                            An example JBoss A-MQ application. For more information about using this temp...   18 (6 blank)      12
amq63-ssl                                       An example JBoss A-MQ application. For more information about using this temp...   16 (6 blank)      10
cakephp-mysql-persistent                        An example CakePHP application with a MySQL database. For more information ab...   20 (4 blank)      9
dancer-mysql-persistent                         An example Dancer application with a MySQL database. For more information abo...   17 (5 blank)      9
datagrid65-basic                                Application template for JDG 6.5 applications.                                     14 (8 blank)      6
datagrid65-https                                Application template for JDG 6.5 applications.                                     23 (13 blank)     8
datagrid65-mysql                                Application template for JDG 6.5 and MySQL applications.                           36 (21 blank)     10
datagrid65-mysql-persistent                     An example JBoss Data Grid application with a MySQL database. For more inform...   37 (21 blank)     11
datagrid65-postgresql                           Application template for JDG 6.5 and PostgreSQL applications built using.          33 (18 blank)     10
datagrid65-postgresql-persistent                An example JBoss Data Grid application with a PostgreSQL database. For more i...   34 (18 blank)     11
datagrid71-basic                                An example JBoss Data Grid application. For more information about using this...   18 (12 blank)     6
datagrid71-https                                An example JBoss Data Grid application. For more information about using this...   27 (17 blank)     8
datagrid71-mysql                                An example JBoss Data Grid application with a MySQL database. For more inform...   40 (25 blank)     10
datagrid71-mysql-persistent                     An example JBoss Data Grid application with a MySQL database. For more inform...   41 (25 blank)     11
datagrid71-partition                            An example JBoss Data Grid application. For more information about using this...   20 (12 blank)     8
datagrid71-postgresql                           An example JBoss Data Grid application with a PostgreSQL database. For more i...   37 (22 blank)     10
datagrid71-postgresql-persistent                An example JBoss Data Grid application with a PostgreSQL database. For more i...   38 (22 blank)     11
datagrid72-basic                                An exampleRed Hat JBoss Data Grid application. For more information about usi...   18 (12 blank)     6
datagrid72-https                                An exampleRed Hat JBoss Data Grid application. For more information about usi...   27 (17 blank)     8
datagrid72-mysql                                An exampleRed Hat JBoss Data Grid application with a MySQL database. For more...   40 (25 blank)     10
datagrid72-mysql-persistent                     An exampleRed Hat JBoss Data Grid application with a MySQL database. For more...   41 (25 blank)     11
datagrid72-partition                            An exampleRed Hat JBoss Data Grid application. For more information about usi...   20 (12 blank)     8
datagrid72-postgresql                           An exampleRed Hat JBoss Data Grid application with a PostgreSQL database. For...   37 (22 blank)     10
datagrid72-postgresql-persistent                An exampleRed Hat JBoss Data Grid application with a PostgreSQL database. For...   38 (22 blank)     11
datavirt63-basic-s2i                            Application template for JBoss Data Virtualization 6.3 services built using S2I.   19 (4 blank)      6
datavirt63-extensions-support-s2i               An example JBoss Data Virtualization application. For more information about...    34 (7 blank)      10
datavirt63-secure-s2i                           An example JBoss Data Virtualization application. For more information about...    50 (20 blank)     8
decisionserver64-amq-s2i                        An example BRMS decision server A-MQ application. For more information about...    30 (5 blank)      10
decisionserver64-basic-s2i                      Application template for Red Hat JBoss BRMS 6.4 decision server applications...    17 (5 blank)      5
decisionserver64-https-s2i                      An example BRMS decision server application. For more information about using...   24 (6 blank)      7
django-psql-persistent                          An example Django application with a PostgreSQL database. For more informatio...   20 (5 blank)      9
eap70-amq-persistent-s2i                        An example EAP 7 A-MQ application. For more information about using this temp...   34 (10 blank)     12
eap70-amq-s2i                                   An example EAP 7 A-MQ application. For more information about using this temp...   32 (10 blank)     11
eap70-basic-s2i                                 An example EAP 7 application. For more information about using this template,...   16 (5 blank)      6
eap70-https-s2i                                 An example EAP 7 application. For more information about using this template,...   26 (11 blank)     8
eap70-mongodb-persistent-s2i                    An example EAP 7 application with a MongoDB database. For more information ab...   39 (18 blank)     11
eap70-mongodb-s2i                               An example EAP 7 application with a MongoDB database. For more information ab...   38 (18 blank)     10
eap70-mysql-persistent-s2i                      An example EAP 7 application with a MySQL database. For more information abou...   40 (19 blank)     11
eap70-mysql-s2i                                 An example EAP 7 application with a MySQL database. For more information abou...   39 (19 blank)     10
eap70-postgresql-persistent-s2i                 An example EAP 7 application with a PostgreSQL database. For more information...   37 (16 blank)     11
eap70-postgresql-s2i                            An example EAP 7 application with a PostgreSQL database. For more information...   36 (16 blank)     10
eap70-sso-s2i                                   An example EAP 7 Single Sign-On application. For more information about using...   44 (19 blank)     8
eap70-third-party-db-s2i                        An example EAP 7 DB application. For more information about using this templa...   30 (7 blank)      8
eap70-tx-recovery-s2i                           An example EAP 7 application. For more information about using this template,...   18 (5 blank)      8
eap71-amq-persistent-s2i                        An example EAP 7 A-MQ application. For more information about using this temp...   35 (11 blank)     12
eap71-amq-s2i                                   An example EAP 7 A-MQ application. For more information about using this temp...   33 (11 blank)     11
eap71-basic-s2i                                 An example EAP 7 application. For more information about using this template,...   17 (5 blank)      6
eap71-https-s2i                                 An example EAP 7 application. For more information about using this template,...   27 (11 blank)     8
eap71-mongodb-persistent-s2i                    An example EAP 7 application with a MongoDB database. For more information ab...   40 (19 blank)     11
eap71-mongodb-s2i                               An example EAP 7 application with a MongoDB database. For more information ab...   39 (19 blank)     10
eap71-mysql-persistent-s2i                      An example EAP 7 application with a MySQL database. For more information abou...   41 (20 blank)     11
eap71-mysql-s2i                                 An example EAP 7 application with a MySQL database. For more information abou...   40 (20 blank)     10
eap71-postgresql-persistent-s2i                 An example EAP 7 application with a PostgreSQL database. For more information...   38 (17 blank)     11
eap71-postgresql-s2i                            An example EAP 7 application with a PostgreSQL database. For more information...   37 (17 blank)     10
eap71-sso-s2i                                   An example EAP 7 Single Sign-On application. For more information about using...   45 (20 blank)     8
eap71-third-party-db-s2i                        An example EAP 7 DB application. For more information about using this templa...   31 (8 blank)      8
eap71-tx-recovery-s2i                           An example EAP 7 application. For more information about using this template,...   19 (5 blank)      8
fuse70-console                                  The Red Hat Fuse Console eases the discovery and management of FIS applicatio...   8 (1 blank)       4
fuse70-console-cluster                          The Red Hat Fuse Console eases the discovery and management of FIS applicatio...   8 (1 blank)       4
jenkins-ephemeral                               Jenkins service, without persistent storage....                                    7 (all set)       6
jenkins-pipeline-example                        This example showcases the new Jenkins Pipeline integration in OpenShift,...       16 (4 blank)      8
jws31-tomcat7-basic-s2i                         Application template for JWS applications built using S2I.                         12 (3 blank)      5
jws31-tomcat7-https-s2i                         An example JBoss Web Server application configured for use with https. For mo...   17 (5 blank)      7
jws31-tomcat7-mongodb-persistent-s2i            An example JBoss Web Server application with a MongoDB database. For more inf...   30 (12 blank)     10
jws31-tomcat7-mongodb-s2i                       Application template for JWS MongoDB applications built using S2I.                 29 (12 blank)     9
jws31-tomcat7-mysql-persistent-s2i              An example JBoss Web Server application with a MySQL database. For more infor...   31 (13 blank)     10
jws31-tomcat7-mysql-s2i                         Application template for JWS MySQL applications built using S2I.                   30 (13 blank)     9
jws31-tomcat7-postgresql-persistent-s2i         An example JBoss Web Server application with a PostgreSQL database. For more...    28 (10 blank)     10
jws31-tomcat7-postgresql-s2i                    Application template for JWS PostgreSQL applications built using S2I.              27 (10 blank)     9
jws31-tomcat8-basic-s2i                         An example JBoss Web Server application. For more information about using thi...   12 (3 blank)      5
jws31-tomcat8-https-s2i                         An example JBoss Web Server application. For more information about using thi...   17 (5 blank)      7
jws31-tomcat8-mongodb-persistent-s2i            An example JBoss Web Server application with a MongoDB database. For more inf...   30 (12 blank)     10
jws31-tomcat8-mongodb-s2i                       Application template for JWS MongoDB applications built using S2I.                 29 (12 blank)     9
jws31-tomcat8-mysql-persistent-s2i              An example JBoss Web Server application with a MySQL database. For more infor...   31 (13 blank)     10
jws31-tomcat8-mysql-s2i                         Application template for JWS MySQL applications built using S2I.                   30 (13 blank)     9
jws31-tomcat8-postgresql-persistent-s2i         Application template for JWS PostgreSQL applications with persistent storage...    28 (10 blank)     10
jws31-tomcat8-postgresql-s2i                    Application template for JWS PostgreSQL applications built using S2I.              27 (10 blank)     9
mariadb-persistent                              MariaDB database service, with persistent storage. For more information about...   9 (3 generated)   4
mongodb-persistent                              MongoDB database service, with persistent storage. For more information about...   9 (3 generated)   4
mysql-persistent                                MySQL database service, with persistent storage. For more information about u...   9 (3 generated)   4
nodejs-mongo-persistent                         An example Node.js application with a MongoDB database. For more information...    19 (4 blank)      9
openjdk18-web-basic-s2i                         An example Java application using OpenJDK 8. For more information about using...   8 (1 blank)       5
postgresql-persistent                           PostgreSQL database service, with persistent storage. For more information ab...   8 (2 generated)   4
processserver64-amq-mysql-persistent-s2i        An example BPM Suite application with A-MQ and a MySQL database. For more inf...   49 (13 blank)     14
processserver64-amq-mysql-s2i                   An example BPM Suite application with A-MQ and a MySQL database. For more inf...   47 (13 blank)     12
processserver64-amq-postgresql-persistent-s2i   An example BPM Suite application with A-MQ and a PostgreSQL database. For mor...   46 (10 blank)     14
processserver64-amq-postgresql-s2i              An example BPM Suite application with A-MQ and a PostgreSQL database. For mor...   44 (10 blank)     12
processserver64-basic-s2i                       An example BPM Suite application. For more information about using this templ...   18 (5 blank)      5
processserver64-mysql-persistent-s2i            An example BPM Suite application with a MySQL database. For more information...    40 (14 blank)     10
processserver64-mysql-s2i                       An example BPM Suite application with a MySQL database. For more information...    39 (14 blank)     9
processserver64-postgresql-persistent-s2i       An example BPM Suite application with a PostgreSQL database. For more informa...   37 (11 blank)     10
processserver64-postgresql-s2i                  An example BPM Suite application with a PostgreSQL database. For more informa...   36 (11 blank)     9
rails-pgsql-persistent                          An example Rails application with a PostgreSQL database. For more information...   21 (4 blank)      9
rhdm70-full-persistent                          Application template for Red Hat Decision Manager 7.0 applications with persi...   30 (7 blank)      11
rhdm70-kieserver                                Application template for Red Hat Decision Manager Execution Server 7.0 applic...   28 (10 blank)     5
rhdm70-kieserver-basic-s2i                      Application template for Red Hat Decision Manager Execution Server 7.0 applic...   24 (7 blank)      5
rhdm70-kieserver-https-s2i                      Application template for Red Hat Decision Manager Execution Server 7.0 applic...   29 (8 blank)      7
rhpam70-authoring                               Application template for a non-HA persistent authoring environment, for Red H...   48 (18 blank)     10
rhpam70-authoring-ha                            Application template for a HA persistent authoring environment, for Red Hat P...   67 (28 blank)     21
rhpam70-kieserver-externaldb                    Application template for a managed KIE server with an external database, for...    46 (26 blank)     5
rhpam70-kieserver-mysql                         Application template for a managed KIE server with a MySQL database, for Red...    44 (20 blank)     8
rhpam70-kieserver-postgresql                    Application template for a managed KIE server with a PostgreSQL database, for...   45 (20 blank)     8
rhpam70-prod                                    Application template for a managed HA production runtime environment, for Red...   59 (25 blank)     26
rhpam70-prod-immutable-kieserver                Application template for an immultable KIE server in a production environment...   48 (19 blank)     10
rhpam70-prod-immutable-monitor                  Application template for a router and monitoring console in a production envi...   38 (17 blank)     10
rhpam70-sit                                     Application template for system integration testing, for Red Hat Process Auto...   59 (25 blank)     26
rhpam70-trial-ephemeral                         Application template for an ephemeral authoring and testing environment, for...    30 (15 blank)     6
s2i-fuse70-eap-camel-amq                        Red Hat Fuse on EAP Camel A-MQ quickstart. This example demonstrates using th...   26 (7 blank)      6
s2i-fuse70-eap-camel-cdi                        Red Hat Fuse on EAP Camel CDI quickstart. This example demonstrates using the...   22 (5 blank)      6
s2i-fuse70-eap-camel-cxf-jaxrs                  Red Hat Fuse on EAP Camel CXF JAX-RS quickstart. This example demonstrates us...   22 (5 blank)      6
s2i-fuse70-eap-camel-cxf-jaxws                  Red Hat Fuse on EAP Camel CXF JAX-WS quickstart. This example demonstrates us...   22 (5 blank)      6
s2i-fuse70-eap-camel-jpa                        Red Hat Fuse on EAP Camel JPA quickstart. This example demonstrates how to co...   35 (13 blank)     8
s2i-fuse70-karaf-camel-amq                      Camel route using ActiveMQ in Karaf container. This quickstart shows how to u...   20 (4 blank)      3
s2i-fuse70-karaf-camel-log                      A simple Camel route in Karaf container. This quickstart shows a simple Apach...   17 (2 blank)      3
s2i-fuse70-karaf-camel-rest-sql                 Camel example using Rest DSL with SQL Database in Karaf container.  This exam...   20 (4 blank)      5
s2i-fuse70-karaf-cxf-rest                       REST example using CXF in Karaf container. This quickstart demonstrates how t...   16 (2 blank)      5
s2i-fuse70-spring-boot-camel                    Spring Boot and Camel QuickStart. This example demonstrates how you can use A...   16 (2 blank)      3
s2i-fuse70-spring-boot-camel-amq                Spring Boot, Camel and ActiveMQ QuickStart. This quickstart demonstrates how...    19 (4 blank)      3
s2i-fuse70-spring-boot-camel-config             Spring Boot and Camel using ConfigMaps and Secrets. This quickstart demonstra...   19 (2 blank)      3
s2i-fuse70-spring-boot-camel-drools             Spring Boot, Camel and JBoss BRMS QuickStart. This example demonstrates how y...   19 (3 blank)      3
s2i-fuse70-spring-boot-camel-infinispan         Spring Boot, Camel and JBoss Data Grid QuickStart. This quickstart demonstrat...   17 (2 blank)      3
s2i-fuse70-spring-boot-camel-teiid              Spring Boot, Camel and JBoss Data Virtualization QuickStart. This example dem...   20 (4 blank)      3
s2i-fuse70-spring-boot-camel-xml                Spring Boot and Camel Xml QuickStart. This example demonstrates how you can u...   16 (2 blank)      3
s2i-fuse70-spring-boot-cxf-jaxrs                Spring Boot and CXF JAX-RS QuickStart. This example demonstrates how you can...    17 (2 blank)      5
s2i-fuse70-spring-boot-cxf-jaxws                Spring Boot and CXF JAX-WS QuickStart. This example demonstrates how you can...    17 (2 blank)      5
sso72-https                                     An example RH-SSO 7 application. For more information about using this templa...   26 (15 blank)     6
sso72-mysql                                     An example RH-SSO 7 application with a MySQL database. For more information a...   36 (20 blank)     8
sso72-mysql-persistent                          An example RH-SSO 7 application with a MySQL database. For more information a...   37 (20 blank)     9
sso72-postgresql                                An example RH-SSO 7 application with a PostgreSQL database. For more informat...   33 (17 blank)     8
sso72-postgresql-persistent                     An example RH-SSO 7 application with a PostgreSQL database. For more informat...   34 (17 blank)     9
sso72-x509-https                                An example RH-SSO 7 application. For more information about using this templa...   12 (6 blank)      4
sso72-x509-mysql-persistent                     An example RH-SSO 7 application with a MySQL database. For more information a...   23 (11 blank)     7
sso72-x509-postgresql-persistent                An example RH-SSO 7 application with a PostgreSQL database. For more informat...   20 (8 blank)      7
```

**Comment by amitkrout** [comment]
> CI failure is not related to this PR.

@praveenkumar right, my intention was to just check if addon-xpaas.feature is working properly or not in CI. Otherwise we need to wait for first nightly test after this is merged. It would be a just a before hand check before running it through CI ```make integration_all``` nightly.

**Comment by agajdosi** [comment]
@amitkrout It was run when PR was opened AFAIR.

**Pull request merged**